index.js

```diff
@@ -85,17 +79,22 @@ function MultiDrive (name, opts, cb) {
       series(fns, function (err) {
         if (err) return done(err)
         var drive = hyperdrive(db)
-        var _opts = (secretKey)
-          ? xtend(self.opts, opts, { secretKey: secretKey })
-          : xtend(self.opts, opts)
-        var archive = drive.createArchive(key, _opts)
-        self.archives[name] = archive
-        done()
+        var __opts = (secretKey)
```

Comment:
redeclaring of `_opts` is a bit confusing - would've assumed `standard` would throw actually; maybs remove `var` here?